ghost/core/core/frontend/helpers/comment_count.js

```diff
@@ -3,23 +3,24 @@ const {labs} = require('../services/proxy');
 
 function commentCount(options) {
     return new SafeString(`
-       <span data-ghost-comment-count="${this.id}" style="display:none">
-           ${options.fn(this)}
-       </span>`
+       <script
+        data-ghost-comment-count="${this.id}"
+        data-ghost-comment-count-empty="${options.hash.empty}"
+        data-ghost-comment-count-singular="${options.hash.singular}"
+        data-ghost-comment-count-plural="${options.hash.plural}"
```

Comment:
Should we add defaults if those parameters are not provided?

Reply:
That's not in the design, and I think doing so would remove our ability to just output a number - which would be useful for a usecase like: `💬 {{comment_count empty="0"}}`